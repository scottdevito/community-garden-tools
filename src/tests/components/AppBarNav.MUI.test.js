import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import { shallow, mount, render } from 'enzyme';
import AppBarNav from '../../components/reusable/AppBarNav.MUI';

// Snapshot test
it('renders correctly', () => {
  const tree = renderer.create(<AppBarNav />).toJSON();
  expect(tree).toMatchSnapshot();
});
