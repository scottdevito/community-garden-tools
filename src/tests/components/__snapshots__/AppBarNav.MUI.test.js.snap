// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="AppBarNav-root-1"
>
  <header
    className="MuiPaper-root-11 MuiPaper-elevation4-17 MuiAppBar-root-2 MuiAppBar-positionStatic-6 MuiAppBar-colorDefault-8"
  >
    <div
      className="MuiToolbar-root-38 MuiToolbar-regular-40 MuiToolbar-gutters-39"
    >
      <h6
        className="MuiTypography-root-42 MuiTypography-h6-59 MuiTypography-colorInherit-71"
      >
        Community Garden Tools
      </h6>
    </div>
  </header>
</div>
`;
